api/envoy/extensions/filters/http/error_response/v3/error_response.proto

```diff
@@ -0,0 +1,89 @@
+syntax = "proto3";
+
+package envoy.extensions.filters.http.error_response.v3;
+
+import "envoy/config/accesslog/v3/accesslog.proto";
+import "envoy/config/core/v3/base.proto";
+import "envoy/config/core/v3/substitution_format_string.proto";
+
+import "google/protobuf/struct.proto";
+import "google/protobuf/wrappers.proto";
+
+import "udpa/annotations/status.proto";
+import "udpa/annotations/versioning.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.extensions.filters.http.error_response.v3";
+option java_outer_classname = "ErrorResponseProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_status).package_version_status = ACTIVE;
+
+// [#protodoc-title: ErrorResponse]
+// Error Response Filter :ref:`configuration overview <config_http_filters_error_response>`.
+// [#extension: envoy.filters.http.error_Response]
+
+// The configuration to filter and change local response.
+message ResponseMapper {
+  // Filter to determine if this mapper should apply.
+  config.accesslog.v3.AccessLogFilter filter = 1 [(validate.rules).message = {required: true}];
+
+  // The new response status code if specified.
+  google.protobuf.UInt32Value status_code = 2 [(validate.rules).uint32 = {lt: 600 gte: 200}];
+
+  // The new error body text if specified. It will be used in the `%LOCAL_REPLY_BODY%`
+  // command operator in the `body_foramt`.
+  config.core.v3.DataSource body = 3;
+
+  config.core.v3.SubstitutionFormatString body_format_override = 4;
+}
+
+// The configuration to customize HTTP error responses read by Envoy.
+message ErrorResponseConfig {
+  // Configuration of list of mappers which allows to filter and change error response.
+  // The mappers will be checked by the specified order until one is matched.
+  repeated ResponseMapper mappers = 1;
+
+  // The configuration to form response body from the :ref:`command operators <config_access_log_command_operators>`
+  // and to specify response content type as one of: plain/text or application/json.
+  //
+  // Example one: plain/text body_format.
+  //
+  // .. code-block::
+  //
+  //   text_format: %LOCAL_REPLY_BODY%:%RESPONSE_CODE%:path=$REQ(:path)%
+  //
+  // The following response body in `plain/text` format will be generated for a request with
+  // local reply body of "upstream connection error", response_code=503 and path=/foo.
+  //
+  // .. code-block::
+  //
+  //   upstream connect error:503:path=/foo
```

Comment:
"upstream connect[ion] error"?

Reply:
i'll update our copy to include this fix